spec/motion-spec/mac_bacon_spec.rb

```diff
@@ -171,18 +170,18 @@ def triggerChange
       before do
         @observable = MockObservable.new
         @started_at ||= Time.now
-        performSelector('triggerChange', withObject:nil, afterDelay:0.5)
+        performSelector('triggerChange', withObject: nil, afterDelay: 0.5)
         wait_for_change @observable, 'an_attribute' do
-          performSelector('triggerChange', withObject:nil, afterDelay:0.5)
+          performSelector('triggerChange', withObject: nil, afterDelay: 0.5)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.